packages/venia-concept/src/drivers/index.js

```diff
@@ -1,5 +1,5 @@
 export { Query } from 'react-apollo';
-export { Link, Redirect, Route } from 'react-router-dom';
+export { Link, Redirect, Route, Switch, withRouter } from 'react-router-dom';
```

Comment:
Consider `export *` here, maybe? This is such a major dependency—I think whitelisting only some of its exports qualifies as a leaky abstraction.

If a consumer of `venia-concept` wanted to import `useRouter()`, for example, they'd encounter the same "conflicting context" issues that this change is addressing.

Reply:
I'd rather export these things one by one to assist in tree shaking.

Reply:
Yeah, I ordinarily wouldn't recommend `export *` for that reason. I think this counts as a serious flaw in this approach, though. If we're not going to `export *`, then we should whitelist all of its exports. And we should consider doing the same for the other third-party libraries being exported here as well.